bokehjs/src/lib/models/tools/inspectors/hover_tool.ts

```diff
@@ -224,7 +227,7 @@ export class HoverToolView extends InspectToolView {
     const fullset_indices = selection_manager.inspectors.get(renderer)!
     const subset_indices = renderer.view.convert_selection_to_subset(fullset_indices)
 
-    if (fullset_indices.is_empty()) {
+    if (fullset_indices.is_empty() && fullset_indices.view == null) {
```

Comment:
Perhaps the added condition should have been actually added to `is_empty()`. There are also other properties that `is_empty()` doesn't consider (e.g. `selected_glyphs`). Or is this a special case of sort?

Reply:
Things are a bit muddled,I suppose.  It's possible to "hit" a glyph like `Patch` or `VArea` without setting any indices at all. Is that empty or not? I guess it's a convention that could use some clarification but I am not sure I would like to tackle it in this PR, I'd be ok with a TODO comment added here mentioning this question for now.